locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgstr "Entre com sua senha para ter acesso as suas palavras"
+
+#: src/components/ModalBackupWords.js:160
+#: src/components/ModalResetAllData.js:77
+msgid "Password*"
+msgstr "Senha*"
+
+#: src/components/ModalBackupWords.js:172 src/components/ModalPin.js:90
+#: src/components/tokens/TokenMelt.js:91
+msgid "Go"
+msgstr "Ir"
+
+#: src/components/ModalBackupWords.js:212
+msgid ""
+"Save the words and never share them. Anyone who has access to them will "
+"control your tokens."
+msgstr ""
+"Salve suas palavras e nunca mostre-as a ninguém. Qualquer pessoa com acesso "
+"a elas terá controle sobre seus tokens."
+
+#: src/components/ModalBackupWords.js:226
+msgid "Ok, I have saved them"
+msgstr "Ok, eu as salvei"
+
+#: src/components/ModalBackupWords.js:255
+msgid ""
+"Please, select the words below in the correct order. This is for your safety "
+"and to ensure you wrote them down correctly."
+msgstr ""
+"Por favor, selecione as palavras na ordem correta. Essa etapa é para sua "
+"segurança, garantindo que você anotou as palavras na ordem certa."
+
+#: src/components/ModalBackupWords.js:270
+msgid "Validate"
+msgstr "Validar"
+
+#: src/components/ModalEditToken.js:95
+msgid "Save"
+msgstr "Salvar"
+
+#: src/components/ModalResetAllData.js:43
+msgid "You must write your password or check that you have forgotten it"
+msgstr "Você deve escrever sua senha ou marcar que você a esqueceu"
+
+#: src/components/ModalResetAllData.js:48
+msgid "Confirm message does not match"
+msgstr "Mensagem de confirmação inválida"
+
+#: src/components/ModalResetAllData.js:56
+msgid ""
+"If you reset your wallet, all data will be deleted, and you will lose access "
+"to your tokens. To recover access to your tokens, you will need to import "
+"your words again."
+msgstr ""
+"Se você resetar sua wallet, todos os dados serão deletados e você irá perder "
+"acesso aos seus tokens. Para recuperar o acesso, você deverá importar suas "
+"palavras novamente."
+
+#: src/components/ModalResetAllData.js:58
+#, javascript-format
+msgid "${ firstMessage } You still haven't done the backup of your words."
+msgstr "${ firstMessage } Você ainda não faz backup das suas palavras."
+
+#: src/components/ModalResetAllData.js:74
+#, javascript-format
+msgid ""
+"If you still wanna do it, we need your password and for you to write down "
+"<strong>'${ CONFIRM_RESET_MESSAGE }'</strong> to confirm the operation."
+msgstr ""
+"Se você ainda deseja fazer isso, nós precisamos que você digite sua senha e "
+"também digite <strong>'${ CONFIRM_RESET_MESSAGE }'</strong> para confirmar a "
+"operação."
+
+#: src/components/ModalResetAllData.js:81
+msgid "Confirm message*"
+msgstr "Mensagem de confirmação*"
+
+#: src/components/ModalResetAllData.js:82
+#, javascript-format
+msgid "Write '${ CONFIRM_RESET_MESSAGE }'"
+msgstr "Escreva '${ CONFIRM_RESET_MESSAGE }'"
+
+#: src/components/ModalResetAllData.js:86
+msgid "I forgot my password"
+msgstr "Eu esqueci minha senha"
```

Comment:
Você confirmou que o reset funciona com a wallet traduzida, já que precisa comparar as frases digitadas?

Reply:
The sentence we compare to check if it's right is written on `src/constants.js`. Maybe we should also translate ther

Reply:
I've just checked and it's being translated there also

Reply:
It is translated and I've tested that it works properly.